React/Fabric/Mounting/ComponentViews/Slider/RCTSliderComponentView.mm

```diff
@@ -62,7 +64,12 @@ - (instancetype)initWithFrame:(CGRect)frame
     _maximumTrackImageResponseObserverProxy = RCTImageResponseObserverProxy(self);
     _thumbImageResponseObserverProxy = RCTImageResponseObserverProxy(self);
 
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
```

Comment:
As with most of this file, shouldn't use block comments for `#if !TTARGET_OS_OSX` lines, only the places we're adding code.